components/Navbar.vue

```diff
@@ -285,15 +294,17 @@ export default class NavbarMenu extends mixins(
 
   onScroll() {
     const currentScrollPosition = document.documentElement.scrollTop
+    const searchBarPosition = document
+      .getElementById('networkList')!
+      .getBoundingClientRect().top
```

Comment:
I think it's incorrect to use element selector because the navbar component would be used in other page, not only the landing page.